src/universal/utils/transit.js

```diff
@@ -57,4 +57,32 @@ export default transit.withExtraHandlers([
       : require('url').parse(value)
     ),
   },
-]);
+];
+
+const modifiedHandlers = handlers.withExtraHandlers(extraHandlers);
+
+const reader = transit.reader('json', {
+  handlers: modifiedHandlers.read,
+  cache: false,
+  // Copied from transit-immutable-js. Without this, Error fails to read.
```

Comment:
can you reference a file, line, and commit hash for where you took this from.

That way when these deps are major'd, we can re-reference